schema.py

```diff
@@ -266,22 +267,61 @@ def is_valid(self, data):
         else:
             return True
 
+    def _raise_error(self, schema_error, message, error_list, data=None,
```

Comment:
I think there are some instances where the `error_list` is `[self._error] + x.errors` and other times where it's just `self._error`. Having it prepended probably won't be a problem, but I just wanna make sure it doesn't break anything or is done in such a way that it causes problems.

Reply:
If `error_list=None` is the default you could do:

```python
error_list = error_list or []
error_list.insert(0, self._error)
```

Reply:
On line 382 and 393, there's
```
raise SchemaError([None] + x.autos, [e] + x.errors)
```
So that's behavior that'd have to be accounted for and implemented. For now, `[None] + x.autos` is being passed as the message argument and `[e] + x.errors` is being passed as the error_list argument to `raise_error` instead of `self._error` during those two instances which is why it's currently set up so `self._error` isn't hard coded into the function

Reply:
Made it so `self._error` is hard coded into `raise_error()` function. However, as a result, `None` had to be passed to `raise_error()` as `message` argument on line 412 and 422